src/generateMongoidTypes.ts

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
```

Comment:
@adongare  Can you take a stab at refactoring this duplicated code into a parent class?

Reply:
Done